src/component.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import NProgress from "nprogress";
+import Router from "next/router";
```

Comment:
If possible, could you investigate the usage of `withRouter` (https://nextjs.org/docs/#using-a-higher-order-component) that will give you the instanced Router, I found some issues when importing next/router causing Router being not instanced yet.